### PR TITLE
feat: remove unused exports

### PR DESCRIPTION
**Pull request by n1ru4l**
This PR removes all the dead and unused code.
Also, it enables the eslint rule for avoiding committing dead code in the future.

### CODE REVIEW

File: packages/web/app/src/components/v2/icon.tsx [hunk 1]
```diff
@@ -149,21 +149,6 @@ export const ArrowDownIcon: FC<IconProps> = ({ className }) => (
   </svg>
 );
 
-export const MailIcon: FC<IconProps> = ({ className, ...props }) => (
```
Comment (B2o5T) [inline packages/web/app/src/components/v2/icon.tsx:152]: Let’s keep icons from this file, we’ll add storybook after with icons stories

**Review by B2o5T — CHANGES_REQUESTED** [review]
Let’s just keep icons from v2/Icon.tsx, also you can setup this rule to explore unused exports https://github.com/kamilkisiela/graphql-config/blob/master/website/.eslintrc.js#L15

File: packages/web/app/src/components/v2/icon.tsx (hunk 1 above)
Comment (n1ru4l) [reply packages/web/app/src/components/v2/icon.tsx:152]: I don't see why we should keep unused stuff within the code-base? What is the point of it?

Comment (B2o5T) [reply packages/web/app/src/components/v2/icon.tsx:152]: I want only keep icons, as they are already dark-mode friendly, we’ll use them in the future

Comment (n1ru4l) [reply packages/web/app/src/components/v2/icon.tsx:152]: When we use them you can just pick them from the git history? I don't see the point in having commented/dead code on the `main` branch..

Comment (B2o5T) [reply packages/web/app/src/components/v2/icon.tsx:152]: Ok remove it

**Review by B2o5T — APPROVED** [review]

**Pull request merged**